### DISCUSSION TITLE
XIAO HMI with display, camera, microphone and speaker

### DISCUSSION THREAD

**Discussion by kevin-seeed** [Seeed-Studio/OSHW-XIAO-Series#3] [Open for Vote]
Create an HMI (Human-Machine Interface) device with a screen using XIAO that supports Arduino, can connect to ILM, and requires a simpler method for screen UI design.
display resolution, size, or XIAO MCU, you decide, please leave comments.

**Comment by CWGwaltney** [comment]
Yes I agree
